docs/source/release_notes.rst

```diff
@@ -24,6 +24,8 @@ Future Release
         * Allow deep EntitySet equality check (:pr:`1480`)
         * Update ``EntitySet.concat`` to work with Woodwork DataFrames (:pr:`1490`)
         * Add function to list semantic tags (:pr:`1486`)
+        * Remove categorical-encoding as an add-on library (:pr:`1632`)
+        * initialize Woodwork on feature matrix in ``remove_highly_correlated_features`` if necessary (:pr:`1618`)
```

Comment:
Not a big deal as I suspect this is all going to be reworked before release anyway, but maybe capitalize `initialize` to be consistent with other entries.